card/src/main/java/com/adyen/checkout/card/CardComponent.kt

```diff
@@ -74,7 +75,8 @@ class CardComponent internal constructor(
 
     override fun requiresInput() = cardDelegate.requiresInput()
 
-    override fun onCleared() {
+    @RestrictTo(RestrictTo.Scope.TESTS)
```

Comment:
`@VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)` for testing would be a bit nicer. Restricting it to testing means it really only should be used in tests and this is actually a protected method

Reply:
Yes indeed, used the wrong annotation 😅